ucum-java8/src/main/java/systems/uom/ucum/format/UCUMFormat.java

```diff
@@ -286,12 +287,26 @@ public Appendable format(Unit<?> unknownUnit, Appendable appendable) throws IOEx
 	    symbol = temp;
 	} else if (unit.getSymbol() != null) {
 	    symbol = unit.getSymbol();
-	} else {
+	} else if (unknownUnit instanceof TransformedUnit) {
+            final StringBuilder temp = new StringBuilder();
+
+            Unit<?> parentUnit = ((TransformedUnit) unit).getParentUnit();
+            UnitConverter converter;
```

Comment:
😀 I just saw the big green block, and I thought, "did he forget all those things we talked about"? haha Sorry about that. I thought you wrote it from scratch.